src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -879,6 +879,35 @@ public function testPutWithAlreadyExistingUrl()
         $this->assertEquals(1103, $response['code']);
     }
 
+    public function testPutWithAuthor()
```

Comment:
Why not adding this to the existing `testPut` test? Would result in less jackrabbit requests, which as we know are likely to break when a lot are coming in on the same process 😕